sampleSDK/sampleSDKApp.swift

```diff
@@ -20,29 +21,29 @@ struct sampleSDKApp: App {
                 print("Error during web view navigation: \(error.localizedDescription)")
             }
         }
-
+        
         // Simple config object
         // expects only your airgap bundle url
-         let transcendCoreConfigSimple: TranscendCoreConfig = TranscendCoreConfig(transcendConsentUrl: "https://transcend-cdn.com/cm/63b35d96-a6db-436f-a1cf-ea93ae4be24e/airgap.js")
-
+        let transcendCoreConfigSimple: TranscendCoreConfig = TranscendCoreConfig(transcendConsentUrl: "https://transcend-cdn.com/cm-test/c7561f1c-7ec9-498c-a401-7219e3b36a8c/airgap.js")
```

Comment:
Which org is this?

Reply:
`Consent Testing` 🙈

Reply:
Cool - should be fine.

Let's remove/update the references to eshopit in this repo? Maybe just use example.com instead?

Reply:
Or update it back to use eshopit, I don't have a strong preference either way

Reply:
removed the references ✅